.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Release tracker
+about: Create an issue to track tasks for a Cluster API version update
+title: Release tracker for v<>
+
+---
+
+/area provider/ibmcloud
+
+**Tasks:**
+
+After every CAPIBM major version release:
+- [ ] [Update version in metadata.yaml](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/metadata.yaml) and [e2e test config files](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/test/e2e/config)
```

Comment:
@Amulyam24 what do we update exactly in e2e test config files post our release?

Reply:
We update the infrastructure provider version. Added the change, PTAL.